plugin.video.twit/addon.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.twit"
+             name="TWiT"
+             version="2.1.8"
+             provider-name="divingmule">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+    <import addon="script.module.feedparser" version="5.2.1"/>
+    <import addon="script.ircchat" version="1.0.1" optional="true"/>
+    <import addon="script.module.simple.downloader" version="1.9.5"/>
+    <import addon="script.module.beautifulsoup4" version="4.6.2"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>audio video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">The TWiT.tv Netcast Network</summary>
+    <summary lang="no">TWiT.tv Netcast-nettverk</summary>
+    <description lang="en">
+      https://twit.tv/about/what-is-twit-tv
+      [CR]The TWiT.tv Netcast Network with Leo Laporte features the #1 ranked technology podcast This Week in Tech, along with over 20 other top-ranked online shows.
+    </description>
+    <description lang="no">
+      https://twit.tv/about/what-is-twit-tv
+      [CR]TWiT.tv Netcast-nettverket med Leo Laporte har den topprangerte teknologi-podkasten «This Week in Tech», sammen med over 20 andre topprangerte online show.
+    </description>
+    <platform>all</platform>
+    <language>en no</language>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
please use a SPDX compliant identifier for the license if possible. See: https://spdx.org/licenses/